src/main/java/com/epam/tc/hw10/services/CrudMethods.java

```diff
@@ -0,0 +1,8 @@
+package com.epam.tc.hw10.services;
+
+public class CrudMethods {
```

Comment:
it is better to name class like TrelloURI , BoardURI , etc.

May be you will create more classes, this will ok as well